examples/v2/chat2.nim

```diff
@@ -70,6 +71,23 @@ proc connectToNodes(c: Chat, nodes: seq[string]) {.async.} =
   await c.node.connectToNodes(nodes)
   c.connected = true
 
+proc selectRandomNode(): string =
+  randomize()
```

Comment:
I don't think it really matters, but I think in Nim `rng` is preferred a la https://github.com/status-im/nim-eth/blob/4e58eb48ce1b1a06d28802ad22215de9ff92a916/tests/p2p/test_routing_table.nim#L7-L18